### DISCUSSION TITLE
中央的鬼天氣

### DISCUSSION THREAD

**Discussion by poiu6fi** [1101-GS4719-C/Python-Community-GS4719-C#2007] [General]
在中央一個禮拜會經歷到一年四季的各種天氣，有時候出大太陽，隔天就下雨變超冷，而且有時候坐公車下山天氣馬上就好起來了，到底是怎麼回事，我們中央是有什麼結界嗎

**Comment by 110101501** [comment]
對，中央可能確實有結界。光是中央所處的地理位置就可以初步論證了，偏遠又天氣多變。
我認為，很多地方可能都還沒辦法體會中央的一週四季，可以推薦親朋好友來中央體驗。

**Comment by yubo0oo** [comment]
同意,整個都很潮濕而且蟲很多,6月了還只有22度,地球壞掉了

**Comment by yayun601506** [comment]
真的~衣櫥裡面春夏秋冬的衣服都有，現在還是大水蟻季，快瘋了

**Comment by 1b5b** [comment]
確實，宿舍都不知被大水蟻攻占了幾次

**Comment by cassy0201** [comment]
真的，衣櫃裡的衣服都不能收....

**Comment by asd99263fg** [comment]
真的超搞，每個禮拜都想往台北跑

**Comment by 710388** [comment]
山上本來就溫差比較大吧，國中地理課本好像是這樣說

**Comment by lilishh** [comment]
好像是第一次在六月感受到冷吧，真的蠻詭異的，而且很多時候天氣預報寫的降雨機率10%，那10%就真的落在中央裡哈哈哈。最近下很多雨就算了，大水螞蟻也超多，只能說中央的天氣真的超鬼。

**Comment by tshsu04224** [comment]
確實天氣很詭異，尤其我就是個完完全全的雨神，天氣超級好的踏出門，隔個五分鐘就開始下雨，真的心情很不美麗🥲

**Comment by wanderer657** [comment]
中央天氣變化快，
一周四季見奇妙。
烈日炎炎照人熱，
驟雨急降轉陰涼。
公車下山即轉晴，
結界奇幻誰能詳。
山中氣象玄妙事，
人間奇觀意難量。

**Comment by 155Julia** [comment]
我覺得版主的想法完全沒錯，中央的天氣永遠跟外界是相反的，天氣預報說的下雨騎過不會下，說的晴朗其實卻是下大雨喔！我也是來到中央才知道，台灣有這種故事神話般的結界就在中央大學。

**Comment by yubo0oo** [comment]
中央的天氣超詭異
